app/services/pub_date_builder.rb

```diff
@@ -26,9 +26,9 @@ def self.pub_date_from_status_primary(event_dates)
   private_class_method :pub_date_from_status_primary
 
   # @return String date.value from a date of type of publication
```

Comment:
Would you mind fixing this yard doc?  I believe it can also return `NilClass`, too, right?

Reply:
feel free to change it.  We did work on this PR together in the beginning.

Reply:
fixed.